### PR TITLE
Add rename / refactor data to update projects for the recent mass renaming

### PR DESCRIPTION
**Pull request by dixonjoel**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md). _(Required)_

### What does this Pull Request accomplish?

Updates the rename refactor tool to account for the renames done recently in vi.lib.

### Why should this Pull Request be merged?

Helps customers that have measurement services to update them for the latest changes.

### What testing has been done?

Ran the refactor tool on the generator template project and it was unbroken after running it.

### CODE REVIEW

File: Source/Generator/_Editor/RenameRefactor/Refactor Dependencies for Projects.vi
Comment (dixonjoel) [inline Source/Generator/_Editor/RenameRefactor/Refactor Dependencies for Projects.vi:1]: image

image

File: Source/Generator/_Editor/RenameRefactor/MeasurementLink Rename Refactor Data.vi
Comment (dixonjoel) [inline Source/Generator/_Editor/RenameRefactor/MeasurementLink Rename Refactor Data.vi:1]: image

Comment (jasonmreding) [inline Source/Generator/_Editor/RenameRefactor/MeasurementLink Rename Refactor Data.vi:1]: For consistency, I would probably rename this to `V3 Upgrade Refactor Data.vi` or `V2 to V3 Upgrade Refactor Data.vi`.

Comment (jasonmreding) [reply Source/Generator/_Editor/RenameRefactor/MeasurementLink Rename Refactor Data.vi:1]: It looks like you are mapping `MeasurementLink Measurement Server Internal.lvlib` to both `ni.measurementlink.measurement.v1.api.lvlib` and `ni.measurementlink.measurement.v2.api.lvlib` which seems wrong. I suspect that means everything ends up getting linked to the v1 APIs?

Comment (dixonjoel) [reply Source/Generator/_Editor/RenameRefactor/MeasurementLink Rename Refactor Data.vi:1]: Probably. I'll fix that - probably a copy-paste oversight.

File: Source/Generator/_Editor/RenameRefactor/Refactor Dependencies for Projects.vi
Comment (jasonmreding) [reply Source/Generator/_Editor/RenameRefactor/Refactor Dependencies for Projects.vi:1]: I would update the ring control text to correspond to the release versions of the repo. I think putting a date in parenthesis behind it is still fine if you are looking to give some context as to when the changes were submitted without the user having to look through the release history.

**Review by jasonmreding — APPROVED** [review]

**Pull request merged**